tests/test_manylinux.py

```diff
@@ -140,6 +140,7 @@ def test_glibc_version_string_ctypes_missing(monkeypatch):
     assert _glibc_version_string_ctypes() is None
 
 
+@pytest.mark.skipif(not ctypes, reason="requires ctypes")
```

Comment:
I wonder if it is better to do xfail here so we can catch this marker if(?) this somehow no longer needs ctypes.

Reply:
https://docs.pytest.org/en/6.2.x/skipping.html

Reply:
```suggestion
@pytest.mark.xfail(ctypes is None, reason="do not have ctypes available")
```